libs/deal-ticket/src/components/deal-ticket/margin-mode-selector.tsx

```diff
@@ -39,7 +40,7 @@ export const MarginChange = ({
   marginMode,
   marginFactor,
 }: {
-  partyId: string | null;
+  partyId: string | undefined;
```

Comment:
(non-blocking):`string | null` made sense however `partyId: string | undefined` is the same with `partyId?: string`

Reply:
This change is only to support the fact that by default pubKey is now undefined and not null.